### DISCUSSION TITLE
[Art competition] Widening and Narrowing

### DISCUSSION THREAD

**Discussion by whuang20226450** [prosyslab-classroom/cs524-program-analysis#277] [Art competition]
![art competition](https://user-images.githubusercontent.com/29110592/204709472-b4824be8-e571-4bbf-9e31-d2e4b2605dc7.png)

## Short Description
For me, [*widening and narrowing*] means [*"yin yang pattern on the surface of the lake and a cat next to it, fiddling the water"*].

## Long Description (up to 3 sentences)
Widening and narrowing are the concepts we used when the abstract semantics is not directly computable. The concepts of widening and narrowing are opposite, yet they complement each other like yin yang. In the painting, the cat represents us and it fiddles the water means that we are trying to design perfect widening and narrowing operators.

**Comment by KihongHeo** [comment]
Good. BTW, why cat? Do you like cat?

**Comment by whuang20226450** [comment]
> Good. BTW, why cat? Do you like cat?

Yes, I am a cat person!
